.github/scripts/ci-test-assertions.sh

```diff
@@ -4,57 +4,53 @@ set -xe
 
 unset JAVA_TOOL_OPTIONS
 
-export MMTK_EXTREME_ASSERTIONS=1
-. $(dirname "$0")/ci-build.sh
+run_subset() {
+    heap_multiplier=$1
 
-cd $OPENJDK_PATH
+    runbms_dacapo2006_with_heap_multiplier fop $heap_multiplier
+    runbms_dacapo2006_with_heap_multiplier luindex $heap_multiplier
+}
 
 # -- SemiSpace --
 export MMTK_PLAN=SemiSpace
```

Comment:
I thought the idea was to move away from using environment variables since OpenJDK has support for passing MMTk options